server/util/outputNormalization.test.js

```diff
@@ -212,6 +201,6 @@ describe('outputsMatch', () => {
     const output2 =
       'W3C website link\nYou are currently on a link. To click this link, press Control-Option-Space.';
 
-    expect(outputsMatch(output1, output2)).toBe(true);
+    expect(outputsMatch(output1, output2)).toBe(false);
```

Comment:
@stalgiag Given the new requirements about keeping punctuation, I think this example should actually fail. I can see how this specific example (`, W3C website link ...`) could be interpreted as possibly truncated output and imo should warrant some additional manual review from a human tester.